bpf_test.go

```diff
@@ -661,3 +661,65 @@ func TestModuleLoadELF(t *testing.T) {
 	checkLookupElement(t, b)
 	checkProgTestRun(t, b)
 }
+
+func TestOptionalLoading(t *testing.T) {
+	var err error
+	kernelVersion, err = elf.CurrentKernelVersion()
+	if err != nil {
+		t.Fatalf("error getting current kernel version: %v", err)
+	}
+
+	dummyELF := "./tests/dummy.o"
+	if kernelVersion > kernelVersion410 {
+		dummyELF = "./tests/dummy-410.o"
+	} else if kernelVersion > kernelVersion48 {
+		dummyELF = "./tests/dummy-48.o"
+	} else if kernelVersion > kernelVersion46 {
+		dummyELF = "./tests/dummy-46.o"
+	}
```

Comment:
Do you still need to upload these tests object files?

Reply:
Yup